build/uuid-redirects.json

```diff
@@ -121,6 +124,7 @@
     "Compendium.pf2e.equipment-effects.Item.kgEOxqF1q4Sy6r97": "Compendium.pf2e.equipment-effects.Item.Effect: Flaming Star - Armor",
     "Compendium.pf2e.equipment-effects.Item.kgotU0sFmtAHYySB": "Compendium.pf2e.equipment-effects.Item.Effect: Eagle Eye Elixir",
     "Compendium.pf2e.equipment-effects.Item.kkDbalYEavzRpYlp": "Compendium.pf2e.equipment-effects.Item.Effect: Antiplague",
+    "Compendium.pf2e.equipment-effects.Item.kkXy5JWC7OTBqTKg": "Compendium.pf2e.equipment-effects.Item.Effect: Insight Coffee",
```

Comment:
Looks like some insight coffee redirects snuck in here. Probably just need to rebase master onto your branch.